---
layout: post
title: Comprehensive miRNA sequence analysis reveals survival differences in diffuse large B-cell lymphoma patients
authors: Emilia L. Lim, Diane L. Trinh, David W. Scott, Andy Chu, Martin Krzywinski, Yongjun Zhao, A. Gordon Robertson, Andrew J. Mungall, Jacqueline Schein, Merrill Boyle, Anja Mottok, Daisuke Ennishi, Nathalie A. Johnson, Christian Steidl, Joseph M. Connors, Ryan D. Morin, Randy D. Gascoyne, Marco A. Marra
year: 2015
ref: Lim et al. Genome Biology. 2015.
journal: Genome Biology
doi: 10.1186/s13059-014-0568-y
image: /images/papers/lim-genome-biology-microrna-dlbcl.png
pub-url: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0568-y
---

# Abstract

**Background**

Diffuse large B-cell lymphoma (DLBCL) is an aggressive disease, with 30% to 40% of patients failing to be cured with available primary therapy. microRNAs (miRNAs) are RNA molecules that attenuate expression of their mRNA targets. To characterize the DLBCL miRNome, we sequenced miRNAs from 92 DLBCL and 15 benign centroblast fresh frozen samples and from 140 DLBCL formalin-fixed, paraffin-embedded tissue samples for validation.

**Results**

We identify known and candidate novel miRNAs, 25 of which are associated with survival independently of cell-of-origin and International Prognostic Index scores, which are established indicators of outcome. Of these 25 miRNAs, six miRNAs are significantly associated with survival in our validation cohort. Abundant expression of miR-28-5p, miR-214-5p, miR-339-3p, and miR-5586-5p is associated with superior outcome, while abundant expression of miR-324-5p and NOVELM00203M is associated with inferior outcome. Comparison of DLBCL miRNA-seq expression profiles with those from other cancer types identifies miRNAs that were more abundant in B-cell contexts. Unsupervised clustering of miRNAs identifies two clusters of patients that have distinct differences in their outcomes. Our integrative miRNA and mRNA expression analyses reveal that miRNAs increased in abundance in DLBCL appear to regulate the expression of genes involved in metabolism, cell cycle, and protein modification. Additionally, these miRNAs, including one candidate novel miRNA, miR-10393-3p, appear to target chromatin modification genes that are frequent targets of somatic mutation in non-Hodgkin lymphomas.

**Conclusions**

Our comprehensive sequence analysis of the DLBCL miRNome identifies candidate novel miRNAs and miRNAs associated with survival, reinforces results from previous mutational analyses, and reveals regulatory networks of significance for lymphomagenesis.